src/nvim/edit.c

```diff
@@ -2600,9 +2556,7 @@ int cursor_up(long n, int upd_topline)
     if (n >= lnum) {
       lnum = 1;
     } else if (hasAnyFolding(curwin)) {
-      /*
-       * Count each sequence of folded lines as one logical line.
-       */
+      // Count each sequence of folded lines as one logical line.
```

Comment:
Have an extra empty line here looks clearer.
```suggestion
      // Count each sequence of folded lines as one logical line.

```